### PR TITLE
fix: welcome message overlap

### LINKED ISSUE #12178: [RC2] chatWelcomeMessageText height
**Describe the bug**
chatWelcomeMessageText has max height 599px? and will overflow with new chat messages.

**To Reproduce**
1. set a long Welcome Message
2. write something in the chat
3. reload page
4. write more in the chat

**Screenshots**
image

**BBB version (optional):**
2.3 RC2

**Desktop (please complete the following information):**
Windows Chrome 90

### LINKED ISSUE #13160: Welcome message not fully show it RTL 2.4 Beta4
Welcome message for RTL languages not show correctly and it seems that don't have enough height. It should be show fully.

1. Just change language overrides for HTML5 to some RTL Languages.
2. Then create and start a meeting with long welcome message.
3. You'll find that welcome message not show completely in chat area.

Keep in mind that the meeting should start with RTL language. Don't create and start meeting with LTR English language and then change the language!

**Expected behavior**
Should show the welcome message completely and clearly.

**Actual behavior**
Welcome message is not complete and show incorrectly.

**Screenshots**
https://imgur.com/a/Ze9YIh2

BBB version: 2.4 Beta 4

Desktop:
 - OS: Windows
 - Browser: Chrome, Safari
 - Version: 92.0.4515.159

### PR DESCRIPTION
**Pull request by ramonlsouza**
### What does this PR do?

Prevents a visual bug with the welcome message not being correctly displayed in specific conditions.

Tested in Firefox with a 1920x1080 23" monitor.

### Sample message 1:

```
وینک قلبی اتعذب فی هواک وینک قلبی العاشق ما ینساک لا جوابات یا ناسینی فیهم شوق و ود وشایلین سلامات کل یوم راح تجینی ترد لقلبی البسمه و للیل النجمات عندما یحب الرجل المرأة بصمت .. یخسر کل شیء حتى هی… وعندما تحب المرأة الرجل بصوت.. تخسر کل شیء حتى هو! ففی الحب الصمت جریمة الرجل والصوت جریمة المرأة…
اربك تكست هو اول موقع يسمح لزواره الكرام بتحويل الكتابة العربي الى كتابة مفهومة من قبل اغلب برامج التصميم مثل الفوتوشوب و الافترايفكتس و البريمير و الافد ميدا كومبوزر و السموك و برامج اخرى كثيرة
```

#### before
Screen Shot 2022-01-07 at 11 40 05

Screen Shot 2022-01-07 at 11 32 00

#### after
Screen Shot 2022-01-07 at 11 32 24

### Sample message 2:

```
Willkommen zu "<b>%%CONFNAME%%</b>"<br><br>Eine kurze BigBlueButton-Einf&uuml;hrung in Form von Video-Tutorials findet sich unter: <a href="event:https://bigbluebutton.org/html5/">https://bigbluebutton.org/html5/</a>.<br><br>Nach M&ouml;glichkeit bitte ein Headset bzw. Kopfh&ouml;rer nutzen, um St&ouml;rger&auml;uche zu vermeiden.
<br>Um via Telefon teilzunehmen %%DIALNUM%% w&auml;hlen und folgende Konferenz-PIN eingeben: %%CONFNUM%% #<br><i>Hinweis: Mittels der Ziffer 0 k&ouml;nnen telefonisch zugeschaltete Personen selbst ihr Mikrofon ein- und ausschalten.</i><br><br><b>Weitere Steuerung bei telefonischer Teilnahme:</b><br><br>Mit 4 l&auml;sst sich das Mikrofon leiser stellen<br> Mit 5 l&auml;sst sich das Mikrofon auf Standard-Lautst&auml;rke stellen<br> Mit 6 l&auml;sst sich das Mikrofon lauter stellen<br><br>Mit * lassen sich sowohl Mikrofon als auch Lautsprecher ein- und ausschalten<br>Mit 0 l&auml;sst sich das Mikrofon ein- und ausschalten<br>
```

#### before

Screen Shot 2022-01-07 at 11 40 33

Screen Shot 2022-01-07 at 11 30 25

#### after
Screen Shot 2022-01-07 at 11 30 49

### Closes Issue(s)
Closes #13160
Closes #12178

### CODE REVIEW

**Comment by antobinary** [comment]
Awesome @ramonlsouza !!

Could you please comment in plain English how often this force function would trigger? The component name is a bit abstract :sweat_smile: 
I.e. if 100 messages are sent, we trigger force X times..
on switch between private<->public
on toggle chat
on window resize etc?

**Comment by ramonlsouza** [comment]
> Awesome @ramonlsouza !!
> 
> Could you please comment in plain English how often this force function would trigger? The component name is a bit abstract 😅 I.e. if 100 messages are sent, we trigger force X times.. on switch between private<->public on toggle chat on window resize etc?

forceCacheUpdate in time-window-chat-item should be called once if it detects that a system message height is different than its container (the bug only happens when the chat is loaded the first time).

****tl;dr:** 1 time if the bug happens, zero times if it does not happen**
